package.json

```diff
@@ -55,8 +55,7 @@
     "generate:migration": "./development/generate-migration.sh"
   },
   "resolutions": {
-    "3box/ipfs/ipld-zcash/zcash-bitcore-lib/lodash": "^4.17.15",
-    "pubnub/superagent-proxy": "^2.0.0"
```

Comment:
Something's funky here. I don't believe that these resolutions work as advertised. If this and the lockfile are to be believed, we've changed nothing about the version of `superagent-proxy` that was installed yet a warning has gone away? What gives?

Reply:
We had been forcing the installation of `superagent-proxy@^2.0.0`, but we didn't ask for the corresponding version of `superagent`. We weren't comprehensive enough in our resolution basically.

Reply:
But this change doesn't change anything about superagent, we're still downloading the same file

Reply:
Yes, that's true. But we never lacked a compatible version of `superagent`, which is probably why everything worked.
The peer dependency warning is shown whenever there's a mismatch between how the dependencies are _declared_, not between what's needed and what we get on disk.